autode/calculation.py

```diff
@@ -211,13 +218,31 @@ def _add_to_comp_methods(self):
         methods.add(f'{string}.\n')
         return None
 
-    def get_energy(self):
+    def get_energy(self) -> Union[PotentialEnergy, None]:
+        """
+        Total electronic potential energy
+
+        Returns:
+            (autode.values.PotentialEnergy): Energy or None
```

Comment:
For Sphinx(?) or can it extract typing info?

Reply:
ah I see. I imagine it can't, but who knows 🤷 
https://pypi.org/project/sphinx-autodoc-typehints/ looks like it may be able to go the other way around